commit b1dfc3019e53cefafb7e23b9a977ec19207b6c5f
Author: Felix Fietkau <nbd@nbd.name>
Date:   Thu Jan 19 14:20:09 2017 +0100

    MIPS: Lantiq: Keep ethernet enabled during boot
    
    commit 774f0c6419bb8f9d83901d33582c7fe3ba6a6cb3 upstream.
    
    Disabling ethernet during reboot (only to enable it again when the
    ethernet driver attaches) can put the chip into a faulty state where it
    corrupts the header of all incoming packets.
    
    This happens if packets arrive during the time window where the core is
    disabled, and it can be easily reproduced by rebooting while sending a
    flood ping to the broadcast address.
    
    Fixes: 95135bfa7ead ("MIPS: Lantiq: Deactivate most of the devices by default")
    Signed-off-by: Felix Fietkau <nbd@nbd.name>
    Acked-by: John Crispin <john@phrozen.org>
    Cc: hauke.mehrtens@lantiq.com
    Cc: linux-mips@linux-mips.org
    Patchwork: https://patchwork.linux-mips.org/patch/15078/
    Signed-off-by: James Hogan <james.hogan@imgtec.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1213_b1dfc3019e53cefafb7e23b9a977ec19207b6c5f.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1213_b1dfc3019e53cefafb7e23b9a977ec19207b6c5f.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
